go/controller/autoscaler/cluster.go

```diff
@@ -30,9 +30,6 @@ type K8sCluster struct {
 // NewK8sCluster create a new instance of K8sCluster.
 func NewK8sCluster(clientset *kubernetes.Clientset) *K8sCluster {
 	return &K8sCluster{
-		NodeCount: 0,
```

Comment:
It's more idiomatic not to initialize values as default values, since the program already do so.